src/eth/provider.ts

```diff
@@ -52,6 +53,7 @@ export async function disconnect(): Promise<void> {
     const requestManager = new RequestManager(await connection.getProvider())
     const account = (await requestManager.eth_accounts())[0]
     await connection.disconnect()
+    AuthServerProvider.deactivate()
```

Comment:
https://github.com/decentraland/decentraland-connect/blob/master/src/ConnectionManager.ts#L111 There is no reason to do this as connection.disconnect calls the provider deactivate function internaly